### PR TITLE
Add Bastin-1999-92 and Zanchi-2022-170

### PR DESCRIPTION
**Pull request by mathildavz**
# Pull request checklist

- [x] add new concept list
- [ ] add new metadata
- [ ] add new Concepticon concept sets
  * [ ] checked whether the new concept(s) can be applied to existing lists with
    `concepticon notlinked --gloss "NEW_GLOSS"`
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

...

### CODE REVIEW

**Comment by LinguList** [comment]
@mathildavz, if you click on DETAILS in the test, that failed, you will see that the error code is about an "invalid column name". This should show up if you run the test on your system before. So question: did you run `concepticon --repos=YOURPATH test` successfully?

**Comment by LinguList** [comment]
@mathildavz, you have still `GLOSS` as a column name in your metadata for at least the second metadata file. This may have caused the error in the test.

**Comment by LinguList** [comment]
That is: Zanchi-2022-170.tsv-metadata.json has this the column `GLOSS`, although it does not occur in the data.

**Comment by mathildavz** [comment]
> @mathildavz, if you click on DETAILS in the test, that failed, you will see that the error code is about an "invalid column name". This should show up if you run the test on your system before. So question: did you run `concepticon --repos=YOURPATH test` successfully?

I did successfully. However, now that I think about it, I changed something about Zanchi-2022-170 and forgot to run it again.

**Comment by mathildavz** [comment]
Thank you for finding the mistake! @LinguList

**Comment by LinguList** [comment]
So was this the problem?

**Comment by LinguList** [comment]
Okay, one more thing that should be done now. I'll explain here.

You have the LINK which is part of the URL, the end, right? So if you re-write the metadata now as:

```
{
        "name": "LINK",
        "datatype": "string",
        "propertyUrl": "dc:relation",
        "valueUrl": "https://hodel.unipv.it/paveda/meanings//{LINK}"
      }
```

This means, our computer code knows that LINK is a URL-Part and that the concatenation of the base URL `https://hodel.unipv.it/paveda/meanings/` with what we find in LINK is creating the new URL we need.

**Comment by LinguList** [comment]
@mathildavz, the tests show still an error, related to the JSON file this time. Can you check on your computer again, that the `concepticon test` does run without errors? If there's an error, please paste it in full here.

**Comment by LinguList** [comment]
One more point: we do not allow spaces in our headers, so can you adjust the header in Zanchi for `ROLE FRAME`, using an underscore, also in the metadata.json?

**Comment by LinguList** [comment]
```
                    {
                        "datatype": "string",
                        "name": "MICROROLES"
                    },
```

This is the error here: the `,` in the end must be removed, since it is the end of the list in JSON:

```
                    {
                        "datatype": "string",
                        "name": "MICROROLES"
                    }
```

**Comment by LinguList** [comment]
Okay. @tarotis and @AnnikaTjuka could you review this contribution? Annika is in joliday this week, but we are not in a hurry and can do this in the beginning of next week then.

**Comment by LinguList** [comment]
And @MuffinLinwist, please also check this contribution and discussion,  so that you can later add the list by sidwell.

**Review by chrzyki — CHANGES_REQUESTED** [review]
I can also help with the review if need be. One small thing: Please don't include the PDFs directly in the commit/PR (I've also noticed that we've got another PDF in `sources/` that shouldn't be there). Rather, include them in `conceptlists.tsv` (`PDF` column) and make me aware of the PDFs that should be included (here, via mail, Mattermost). I'll then place them on our dedicated storage space for binary files.

**Comment by LinguList** [comment]
Thanks, @chrzyki, we should at some point make sure that I could also upload pdfs if need be, so we do not need to ask you every time. And, @mathildavz, please make a "git rm sources/Bastin1999.pdf" and "git commit" and send the pdf to Christoph in mattermost.

**Comment by LinguList** [comment]
And if you have time to check, Christoph, this is also very welcome of course ;)

**Comment by Tarotis** [comment]
I'll happily provide my first review. Thanks Mathilda for adding the two concept lists.

Regarding the .bib-file, I have two comments:

- Bastin1999 is a book, not an article, so the entry type should be changed
- Zanchi2022 has an URL that can be added: https://aclanthology.org/2022.sigtyp-1.14

In the Bastin concept list, there is one typo and one change I would suggest:

-  Bastin-1999-92-52: `moutain` should probably be `mountain`
- Bastin-1999-92-61: I would propose to change the mapping from `667 ROAD to `2252 PATH `, as this seems more in line with the original description

Further, I have the following suggestions for the Zanchi-2022 list:

- Zanchi-2022-170-52 `feel cold`: currently unmapped, I suggest `3819 COLD (FEELING COLD)`
- Zanchi-2022-170-66 `burn`: currently mapped to `1428 BURNING`; I suggest a change to `2102  BURN`, which seems to be used more frequently and fits the description at least equally
- Zanchi-2022-170-102 `cram`: I actually had to google this first as I didn't knew what this meant, but would `2006  FILL` be a fitting link?
- Zanchi-2022-170-109 `feel sleepy`: currently unmapped, I would suggest `1757 TIRED`
- Zanchi-2022-170-118 `assassinate`: would `KILL` or `MURDER` be adequate?
- Zanchi-2022-170-140 `sink-tr`: currently unmapped, should be `1088 SINK (DESCEND)`
- Zanchi-2022-170-142 `roll-tr`: currently unmapped, should probably be `827 ROLL`
- Zanchi-2022-170-149 `get excited`: currently unmapped, should be `3972 EXCITED`

There are two entries which feature with two different meanings:

- Zanchi-2022-170-125 `come` and `come (2)`
- Zanchi-2022-170-162 `shave (something/somebody` and `shave (self)`

In both cases, I would argue that the concept behind the meaning is the same, and that the difference is merely syntactic. Would you agree?

Possible concept additions I was surprised not to find in the list yet:

- Zanchi-2022-170-113 `capsize`
- Zanchi-2022-170-115 `say (impersonal)`; as `CITE` or `quote`?

**Comment by LinguList** [comment]
Thanks, @Tarotis. Very useful comments.

For sink and similar cases, transitivity is important. And sink in Concepticon is intransitive, so I would not map it. Similar care should be taken for the other concepts.

**Comment by LinguList** [comment]
Burn in Concepticon has a general reading, not specified with resp to transitivity. Burning is intransitive. So mapoing depends from the gloss and the usage.

**Comment by LinguList** [comment]
For cold feel cold I suggest to check which concepts are linked so far. It may just be linked to a concept set "freeze" if we have this and they do not have freeze in their conceptlist already.

**Comment by Tarotis** [comment]
The concepticon description for `sink` includes the following part: `or to cause something to go down in it.`. I would interpret this as transitivity.

**Comment by LinguList** [comment]
Did you check examples? Or broader/narrower? This is more important than the definition, which may be problematic at times. And don't we have another sink in the same list?

**Comment by LinguList** [comment]
But this just needs to be checked and then we see. I am writing from mobile and cannot check right now

**Comment by LinguList** [comment]
@mathildavz and @Tarotis, I propose some checks on the suggestions that I consider critical and should be checked, also illustrating how checking in Concepticon is usually carried out.


> Zanchi-2022-170-52 feel cold: currently unmapped, I suggest 3819 COLD (FEELING COLD)

If you check https://concepticon.clld.org/values, and search for `feel cold`, there is one entry, linked to `<NA>`, that is, it is left empty. We can make a new concepticon gloss for it, but I would not link to "cold (feeling cold)" now.

>    Zanchi-2022-170-66 burn: currently mapped to 1428 BURNING; I suggest a change to 2102 BURN, which seems to be used more frequently and fits the description at least equally

[BURN](https://concepticon.clld.org/parameters/2102) is the least specified notion. But BURNING is explicitly intransitive, and the Role Frame by Zanchi et al. is *s burns*, so I would leave the mapping.

>    Zanchi-2022-170-102 cram: I actually had to google this first as I didn't knew what this meant, but would 2006 FILL be a fitting link?

Probably not, the role frame in Zanchi is *X crams Y into Z*.

>  Zanchi-2022-170-109 feel sleepy: currently unmapped, I would suggest 1757 TIRED

TIRED is an adjectival reading in Concepticon (meaning, it is a state), feel sleepy is a verb. We have one other "feel sleepy" (https://concepticon.clld.org/values/Nagano-2013-1256-27) which is also unlinked so far. So it should not be linked, rather we can later make a new Concepticon Gloss.

>    Zanchi-2022-170-118 assassinate: would KILL or MURDER be adequate?
KILL definitely not, because it occurs in the list already and we do not want to link different rows in a concept list to the same concepticon gloss (only exceptional cases). MURDER can be discussed, but one should check in https://concepticon.clld.org/values if we have similar links in the past.

>   Zanchi-2022-170-140 sink-tr: currently unmapped, should be 1088 SINK (DESCEND)

If you check again the values linked to this concept set, you can see that it is always intransitive, and even more important, we have linked the entry "sink" in Zanchi et al. already to this concept set and want to avoid multiple links. So I'd argue: we should update the definition, as it is misleading, and we should correct [this entry](https://concepticon.clld.org/values/Zalizniak-2020-2590-1133), which is also wrong, as it is explicitly transitive. We can make a new concepticon gloss "SINK (SOMETHING)" and link both to this entry. We should make an issue to discuss this.

>    Zanchi-2022-170-142 roll-tr: currently unmapped, should probably be 827 ROLL

Intransitive "roll" in Zanchi et al. is already mapped to this concept. So rather leave it and check links actively (https://concepticon.clld.org/values/, important to check there whenever in doubt).

>    Zanchi-2022-170-149 get excited: currently unmapped, should be 3972 EXCITED

Adjective vs. verb is not trivial, again, checking https://concepticon.clld.org/values/ is essential, and checking list-internally is also important.

**Comment by mathildavz** [comment]
I propose to change the following (for now) based on your helpful comments @Tarotis and your remarks @LinguList: 

- [ ] change the .bib-file: Bastin a book 
- [ ] change the .bib-file: add URL Zanchi 
- [ ] change Bastin-1999-92-52: `moutain` to `mountain`
- [ ] change Bastin-1999-92-61: `667 ROAD` to `2252 PATH`

I would not map `assassinate` to `MURDER` as the instances in the Concepticon seem to be nouns (the murder, homicide,…), while we are looking at a verb.

Same goes for `get excited´ to `3972 EXCITED`: I believe the instances in the Concepticon are adjectives and not verbs.

**Comment by Tarotis** [comment]
Thank you for the useful comments, I see that I should pay more attention to the examples and not rely on the descriptions as I did previously.

**Comment by LinguList** [comment]
The discussion will go on when Annika joins next week.

File: concepticondata/conceptlists/Bastin-1999-92.tsv [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+Bastin-1999-92-32	32	good	bon	1035	GOOD
+Bastin-1999-92-33	33	ground (on the)	par terre		
+Bastin-1999-92-34	34	hair (of head)	cheveu	2648	HAIR (HEAD)
+Bastin-1999-92-35	35	head	tête	1256	HEAD
+Bastin-1999-92-36	36	hear	entendre	1408	HEAR
+Bastin-1999-92-37	37	heart	coeur	1223	HEART
+Bastin-1999-92-38	38	horn	corne	1393	HORN (ANATOMY)
+Bastin-1999-92-39	39	kill	tuer	1417	KILL
+Bastin-1999-92-40	40	knee	genou	1371	KNEE
+Bastin-1999-92-41	41	know	savoir	3626	KNOW
```
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Bastin-1999-92.tsv:42]: Other instances of _savoir_ are mapped to 410 KNOW (SOMETHING).

File: concepticondata/references/references.bib [hunk 2]
```diff
@@ -1,4 +1,26 @@
 
+@Inproceedings{Zanchi2022,
+  Title                               = {PaVeDa - Pavia Verbs Database: challenges and perspectives},
+  Author                              = {Zanchi, Chiara and Luraghi, Silvia and Combei, Claudia R.},
```
Comment (AnnikaTjuka) [inline concepticondata/references/references.bib:4]: I'd either use Claudia R. also in conceptlists.tsv or Claudia Roberta here.

File: concepticondata/references/references.bib [hunk 3]
```diff
@@ -1,4 +1,26 @@
 
+@Inproceedings{Zanchi2022,
+  Title                               = {PaVeDa - Pavia Verbs Database: challenges and perspectives},
```
Comment (AnnikaTjuka) [inline concepticondata/references/references.bib:3]: We usually use uppercase after a colon.

File: concepticondata/references/references.bib [hunk 4]
```diff
@@ -1,4 +1,26 @@
...
+  Booktitle                           = {Proceedings of the 4th Workshop on Computational Typology and Multilingual NLP},
+  Eventtitle                          = {SIGTYP 2022},
+  Publisher                           = {Association for COmputational Linguistics}, 
+  Address                             = {Seattle},
+  Pages                               = {99-102},
+  URL                                 = {https://aclanthology.org/2022.sigtyp-1.14}
+}
+
+@Book{Bastin1999,
+  Title                               = {Continuity and divergence in the {B}antu languages: perspectives from a lexicostatistic study},
```
Comment (AnnikaTjuka) [inline concepticondata/references/references.bib:15]: Uppercase after colon

File: concepticondata/conceptlists/Zanchi-2022-170.tsv [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+Zanchi-2022-170-43	43	SIT DOWN	sit-down	S sits down (somewhere (L))	sitdowner, sitting-down place, sit down causer	1649	SIT DOWN
+Zanchi-2022-170-44	44	RUN	run	A runs	runner, running place, run causer, run goal, run from source, run distance, running opponent	1519	RUN
+Zanchi-2022-170-45	45	JUMP	jump	A jumps	jumper, jump beneficiary, jump causer, jump instrument, jump location, jumped over (obstacle)	1829	JUMP
+Zanchi-2022-170-46	46	COME (1)	come-1	S comes	comming person	1446	COME
+Zanchi-2022-170-47	47	SING	sing	S sings	singer, song, sing causer, sung-to person	1261	SING
+Zanchi-2022-170-48	48	LIVE	live	S lives somewhere (L)	liver, living place, live causer, life	363	DWELL (LIVE, RESIDE)
+Zanchi-2022-170-49	49	APPEAR	appear	S appears	appearer, appear beneficiary, appear causer, appear location	1076	EMERGE (APPEAR)
+Zanchi-2022-170-50	50	DIE	die	S dies	dieer, killer/death causer, cause of death, dying location	1494	DIE
+Zanchi-2022-170-51	51	FALL	fall	S falls	fallee, fall causer, fall instrument, fall goal, fall source	1280	FALL
+Zanchi-2022-170-52	52	FEEL COLD	feel-cold	S is cold	freezing person, feeling cold locus, chill causer
```
Comment (AnnikaTjuka) [inline concepticondata/conceptlists/Zanchi-2022-170.tsv:53]: We have 3819	COLD (FEELING COLD), but it is categorized as a property.

**Review by AnnikaTjuka — COMMENTED** [review]
Great work! I only left a few more comments.

**Comment by LinguList** [comment]
Thanks @AnnikaTjuka, so I suggest @mathildavz wraps this up, incorporating all of your comments (with which I agree) tomorrow, and we can merge this PR.

**Comment by AnnikaTjuka** [comment]
@chrzyki Since you requested changes, @mathildavz will also need your approval to merge the PR.

**Review by LinguList — APPROVED** [review]

**Comment by LinguList** [comment]
@chrzyki, have your requested changes been addressed? If so, I'd merge this now.

**Comment by LinguList** [comment]
I just saw that you indeed followed the requests, @mathildavz. So I merged right now.

**Pull request merged**